com.chenyifaer.FaForever.yml

```diff
@@ -0,0 +1,82 @@
+id: com.chenyifaer.FaForever
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: faforever
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-download
```

Comment:
you say it's a music player. I don't see anything for audio playback.

Reply:
Is there some value missing in `finish-args`?

Reply:
you tell me. But if this is supposed to play sound, then you likely never tested the package.